yarn.lock

```diff
@@ -1943,7 +1943,7 @@
 
 "@babel/plugin-proposal-private-methods@^7.18.6":
   version "7.18.6"
-  resolved "https://registry.yarnpkg.com/@babel/plugin-proposal-private-methods/-/plugin-proposal-private-methods-7.18.6.tgz#5209de7d213457548a98436fa2882f52f4be6bea"
+  resolved "https://registry.npmjs.org/@babel/plugin-proposal-private-methods/-/plugin-proposal-private-methods-7.18.6.tgz#5209de7d213457548a98436fa2882f52f4be6bea"
```

Comment:
Could this be kept as yarnpkg.com?

Reply:
As the top line states, this file is automatically generated and should not be modified by hand.

Reply:
Sorry, as in can the method used which updated be fixed so that it stays yarnpkg? Has the package been removed from the yarn registry?

Reply:
It sounds a bit out of topic for this PR maybe?

Reply:
We have 1236 occurrences on `master` like this

<img width="324" alt="Screenshot 2024-08-06 at 22 05 08" src="https://github.com/user-attachments/assets/5a36c72b-2d73-4a9e-b947-e67687be249b">

And 2080 in the other case


<img width="326" alt="Screenshot 2024-08-06 at 22 06 16" src="https://github.com/user-attachments/assets/b1e520de-34fd-4cb4-be85-5b0cd70478bd">

Reply:
The PR contains the change, so it's very much in topic.

There's 64 references in `yarn.lock` to `npmjs`, 52 are from `@aws-sdk`.

There's 1167 references in `pkg/eks/yarn.lock` which is a whole other story. It looks like that suffered the same issue as this change.

I've checked and `https://registry.yarnpkg.com/@babel/plugin-proposal-private-methods/-/plugin-proposal-private-methods-7.18.6.tgz#5209de7d213457548a98436fa2882f52f4be6bea` does exist. 

I've tested in a new project and `yarn add @babel/plugin-proposal-private-methods` creates a yarn lock entry containing `yarnpkg.com`.

I'm not sure there's any reason for the change?

Reply:
As I said, `yarn.lock` is automatically generated.
When you have countless conflicts on rebase, the only way is to create a new one.

Is there any specific issue forcing you to block this automatic generation? If not, please it would be great to proceed working on the migration.
We have changes on it constantly: https://github.com/rancher/dashboard/commits/master/yarn.lock

Reply:
So you're saying you had to regenerate yarn.lock in order for the changes in this pr to work, and that regeneration added the change? I couldn't reproduce. Also there's no other changes in this file, so i'm not sure what the conflict was.

This file does change, but 21 times in 7 months though.

From my point of view the PR introduces the change and the logic is not clear how it was brought it or needed.

At the end of the day the PR is approved and can be merged. I'll try and find some time tomorrow to understand what caused this specific issue.

Reply:
FYI I regenerate `yarn.lock` every time. 
This is since I fixed the manually edited version we had 2 years and a half ago and that is what I have done since then, which is what makes it possible to edit the dependencies now or generate a new one.

You are free to investigate the registry, although at the current state and without any research I have no suggestion for you about this.